HunterPie/Game/Rise/Data/AbnormalityData.xml

```diff
@@ -368,6 +403,12 @@
                      Offset="1DC"
                      Name="ABNORMALITY_LATENTPOWER"
                      Icon="ICON_ARMORSKILL_SALMON"
+                     Category="Debuffs"/>.
+        <Abnormality Id="ABN_WINDMANTLE"
+                     Offset="77C"
+                     Name="ABNORMALITY_WINDMANTLE"
+                     Icon="ICON_ARMORSKILL_GREEN"
+                     MaxTimer="15"
```

Comment:
What's the point of this `MaxTimer`? `MaxTimer` of an abnormality is automatically set as `Math.Max(currentTimer, lastTimer);`

Reply:
WindMantle's timer is an off timer. It starts at 0. A timer is set for each weapon, but all are 15 seconds in common.